### DISCUSSION TITLE
Addition of an Axe

### DISCUSSION THREAD

**Discussion by jasper-one** [EveronLife/EveronLife#84] [Show and tell]
Arma Reforger has no pure melee weapons out of the box, so I thought we should add one. I choose a one-handed hatchet/axe because it can be easily mapped to the pistol (@thebonbon discovered & demonstrated this).

![image](https://user-images.githubusercontent.com/106525617/171985497-04ece3fe-ff76-425f-b0fc-8c6ccc13d24f.png)

This also provides a tool to cut wood for #74 once the details are worked out as it is damage-based.

**Comment by Arkensor** [comment]
What are the file sizes on this after import (xob + edds + materials)?

**Reply by jasper-one** [reply]
I managed to get the .xob down to 580 Polys while still looking good so it's just 17kb. 

The .edds is was a little harder. Being a melee weapon it really is up in your face and I can start to see it deteriorate in quality below 512x512. The input file is just 46.1kb JPG -> 280kb TIFF -> 599kb edds. Happy to take the loss and decrease it further if that's going to cause an issue though. 

So all up the item is ~616 kb.

**Reply by Arkensor** [reply]
I think that is alright :) You can turn this into an issue to make a PR for it :)
![image](https://user-images.githubusercontent.com/8494013/171998020-263edfbb-49f6-44f3-b99e-381377997874.png)
